utils/search_tools.py

```
@@ -16,7 +16,14 @@
 
 
 class SearchTool:
-    def __init__(self, filename: str, title: str = "", author: str = "", keywords: str = "", region_override: str = ""):
+    def __init__(
```

Ruff (0.15.0)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)